params/query.yml

```diff
@@ -62,6 +62,10 @@ file_types:
   engine: gromacs
   keywords: none
   category: trajectory
+- type: tng
+  engine: gromacs
+  keywords: none
```

Comment:
I did a quick check and could not find any files for tng. therefore, I would use a more conservative search strategy and add some keywords (e.g. generic_keywords)